src/component/common/ResourceCreationButton/ResourceCreationButton.tsx

```diff
@@ -0,0 +1,18 @@
+import PermissionButton from '../PermissionButton/PermissionButton';
+
+interface IResourceCreationButtonProps {
+    ressourceName: string;
+    permission: string;
+}
+
+export const ResourceCreationButton = ({
```

Comment:
Maybe this component could be called `CreateButton`, and the other one `UpdateButton`?

Reply:
I will go with `CreateButton` and `SaveButton`. Wdyt @olav ?

Reply:
Create/Save could be taken to mean the same thing, so Create/Update works better for me at least. :)